core/src/apps/cardano/layout.py

```diff
@@ -215,16 +215,15 @@ async def confirm_sending(
     ctx: wire.Context,
     ada_amount: int,
     to: str,
-    is_change_output: bool,
+    output_message: str,
```

Comment:
Hm, perhaps we can use `address` instead of `third-party`? But I don't really have an issue with `third-party` 🤔 As that's what we generally use to talk about outputs with addresses.

Reply:
> But I don't really have an issue with third-party :thinking: As that's what we generally use to talk about outputs with addresses.

Yes, but device-owned outputs in Plutus tx are not given by address bytes, but address parameters (like change outputs), they are not `third-party` in any sense.

But `address` sounds good to me :+1:

Reply:
Implemented in https://github.com/vacuumlabs/trezor-firmware/pull/78/commits/34f2cf4611ec15eb3565b7096a5dd690e23c5413.